job_import.py

```diff
@@ -21,12 +23,56 @@
     with open("/sheepdog-creds.json") as pelican_creds_file:
         sheepdog_creds = json.load(pelican_creds_file)
 
-    DB_URL = "jdbc:postgresql://{}/{}".format(
-        sheepdog_creds["db_host"], sheepdog_creds["db_database"]
-    )
+    if "guid" in input_data_json:
+        print("a guid was supplied to the job")
+        print("we are getting a signed url for the given guid")
+
+        host = "https://" + hostname
```

Comment:
Replace using the `hostname` with querying the service directly, similar to https://github.com/uc-cdis/pelican/blob/master/job_export.py#L33

Then it would communicate directly inside the Kubernetes, without going through Squid proxy.

Reply:
changed!